apps/journeys/src/components/Conductor/Conductor.tsx

```diff
@@ -133,7 +139,19 @@ const Conductor = ({ blocks }: ConductorProps): ReactElement => {
                   }
                 }}
               >
-                <BlockRenderer {...block} />
+                {activeBlock?.id === block.id ||
+                previousBlocks[previousBlocks.length - 1]?.id === block.id ? (
+                  <BlockRenderer {...block} />
+                ) : (
+                  <CardWrapper
+                    id={block.id}
+                    backgroundColor={theme.palette.background.paper}
```

Comment:
```suggestion
                    backgroundColor={theme.palette.primary.light}
```
Reverting back to this - I've just remembered we did this so the next Card was more visible